GitLab/README.md

```diff
@@ -0,0 +1,38 @@
+# Nimbella Commander GitLab Command Set
+A Nimbella Commander command set for viewing GitLab project statistics, creating tickets, and assigning issues. Currently supports slack.
+
+## Features
+- View community contribution stats for GitLab projects
+- Create a ticket for a GitLab project
+- Assign issues for a GitLab project
+- View list of users in project repository
+
+## Install
+```
+/nc csm_install xxx
```

Comment:
Should be `/nc csm_install gitlab`?

Reply:
That's a placeholder. I'm wasn't sure what it should be since it's not on my GitHub account. On my personal account, it would be something like 'github:Maljean/GitLab'. I'm  guessing, for now, it would look like 'github:nimbella/command-sets/tree/dev/gitlab'